src/logging/drain.rs

```diff
@@ -0,0 +1,59 @@
+use anyhow::Result;
+use slog::{OwnedKVList, Record};
+
+use super::event;
+
+/// A logging drain designed to integrate with [`slog::Drain`]
+///
+/// The drain can be easily added to a regular [`slog::Logger`]:
+///
+/// ```rust
+/// use kubewarden_policy_sdk::logging;
+/// use slog::{Drain, Logger, o};
+///
+/// let drain = logging::KubewardenDrain::new().fuse();
+/// let log = Logger::root(drain, o!("logger_key1" => "logger_value1"));
+/// ```
+///
+/// The drain behaves differently based on the target architecture used at
+/// build time.
+///
+/// The `wasm32` target architecture will cause the drain to use the [`wapc_guest::host_call`]
+/// helper to propagate the log event from the Wasm guest to the native host
+/// environment. This is the default behaviour for Kubewarden policies at
+/// execution time.
+///
+/// Building for a non `wasm32` architecture will cause the drain to print the log
+/// entries on the standard output.
+/// This is useful for running test of policies via a regular `cargo test`.
+
+#[derive(Default)]
+pub struct KubewardenDrain {}
+
+impl KubewardenDrain {
+    pub fn new() -> KubewardenDrain {
```

Comment:
I think I would remove this method unless you really want to keep it around, I would rely on `KubewardenDrain::default()` on the SDK user side.

Reply:
I'll keep it around, because now we call `fuse` inside of it